### PR TITLE
New change alert monitor guide

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
  - Add new file for change alert guide
  - Add new file to the guide index
  - Add change alert example images

### Motivation
[DOCS-5683](https://datadoghq.atlassian.net/browse/DOCS-5683)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5683]: https://datadoghq.atlassian.net/browse/DOCS-5683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/monitors/guide/change-alert.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+| Option   | Description                                                        | Formula              |
+| -------  | ------------------------------------------------------------------ | -------------------- |
+| Change   | The absolute change of the value.                                  | `a - b`              |
+| % Change | The percentage change of the value compared to its previous value. | `((a - b) / b) * 100`|
+
+In both cases, `Change`, and `% Change` can be either positive or negative. 
+
+## Troubleshooting a change alert evaluation
+
+Change alerts can be mistaken to be false positives. You can reconstruct the metric queries with Notebooks to verify the results of your monitor evaluation.
```
Comment (buraizu) [inline content/en/monitors/guide/change-alert.md:56]:
```suggestion
If you suspect that your change alert evaluation is a false positive, you can reconstruct the metric queries with a Notebook to verify the results. 
```

File: content/en/monitors/guide/_index.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ cascade:
     {{< nextlink href="monitors/guide/why-did-my-monitor-settings-change-not-take-effect" >}}Monitor settings changes not taking effect{{< /nextlink >}}
     {{< nextlink href="monitors/guide/integrate-monitors-with-statuspage" >}}Integrate monitors with Statuspage{{< /nextlink >}}
     {{< nextlink href="monitors/guide/github_gating" >}}Gating your GitHub Actions Deployments with Datadog Monitors{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/change-alert" >}}Change Alert monitors{{< /nextlink >}}
```
Comment (buraizu) [inline content/en/monitors/guide/_index.md:43]: Optional suggestion here, but I think grouping this with the `Anomaly monitors` entry makes slightly more sense for organization

**Review by buraizu — CHANGES_REQUESTED** [review]
Nice work! This review fills in a bit of missing alt text and other minor punctuation details, but I also suggested some alternate phrasing in a couple of places. I'm open to iterating on any of those parts further if you have better suggestions.
Also, just wanted to confirm if it's intentional to include the `change_alert_define_the_metric_options.png` image, as it doesn't seem to be used in the doc.

**Comment by estherk15** [comment]
@buraizu Thanks so much for the review and catching the unused image!

**Review by buraizu — APPROVED** [review]
@estherk15 thanks, LGTM!

**Pull request merged**